lib/index.js

```diff
@@ -44,7 +44,8 @@ import {urlToPath, isUrl} from './minurl.js'
 // Order of setting (least specific to most), we need this because otherwise
 // `{stem: 'a', path: '~/b.js'}` would throw, as a path is needed before a
 // stem can be set.
-const order = ['history', 'path', 'basename', 'stem', 'extname', 'dirname']
+// Note: extname and dirname automatically run last
```

Comment:
This comment is not formulated correctly. Anything in `options` not in this list “automatically” runs last, although it is in fact rather manually, there’s code, after the loop, to do that.

It is important that `extname` and `dirname` run after `history`, `path`, `basename`, and `stem`. Explicitly, as they have been amongst their peers, or implicitly by removing them.

I think it’s rather weird that not all special path-related props are listed now. This can be changed by changing the variable name and the whole comment.